app/controllers/api/v1/orders_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Api::V1::OrdersController < ApplicationController
+
+  def index
+    orders = Order.all
+
+    orders = orders.by_status(params[:status]) if params[:status].present?
+
+    render json: orders, each_serializer: OrderSerializer
+  end
+
+  def show
+    order = Order.find(params[:id])
+
+    render json: order, serializer: OrderSerializer
+  end
+
+  def create
+    order = Order.new(description: params[:data][:attributes][:description])
+
+    order.save!
+
+    render json: order, serializer: OrderSerializer
```

Comment:
you shouldn't need to specify the serializer as long as it follows the naming conventions, it should work without `, serializer: OrderSerializer` :-)